bin/create.js

```diff
@@ -302,26 +337,76 @@ function handleCreate (argv) {
                 }
             })
             .then(function (profile) {
-                return profile.createToken(argv)
+                var promises = [];
+                var unnamed_url = profile.url + '/api/run/'
+                    + (argv.container || profile.container);
+
+                if (argv.name)
+                    var named_url = profile.url
+                        + '/api/run/'
+                        + (argv.container || profile.container)
+                        + '/' + argv.name;
+
+                if(argv.share) {
+                    var aud = named_url.replace('https:\/\/', '');
+
+                    argv.secret.WEBTASK_JWT_SECRET = getSecret();
+
+                    // This is the token we attach to the URL for the user
+                    argv.share = getAuthToken({ aud: aud }, argv.secret.WEBTASK_JWT_SECRET);
+
+                    // Also replace the shared clientId with the URL (the share method is not Auth0-based)
+                    argv.param.WEBTASK_JWT_AUD = aud
+                }
+
+                if(argv.auth0) {
+                    var actual_name = argv.name + '_auth0';
+                    var lock_webtask = {
+                        name: argv.name,
+                        code: Fs.readFileSync(__dirname + '/lib/lock_webtask.js', 'utf8'),
+                        param: {
+                            baseUrl: profile.url,
```

Comment:
Isn't his overridable per `wt create`? it used to be that profile only provided defaults, but all properties could be specified explicitly on create. This was supposed to make scripting easy.